ta4j-core/src/main/java/org/ta4j/core/num/PrecisionNum.java

```diff
@@ -213,6 +216,9 @@ public static PrecisionNum valueOf(BigDecimal val, int precision) {
     /**
      * Returns a {@code Num} version of the given {@code double}.
      *
+     * <p>
```

Comment:
Please apply correct formatting: `<p>` is not necessary. Please also apply asterisks in a convenient way (in convention applied in other javadocs)